.github/dockerfiles/docker_tag

```diff
@@ -1 +1 @@
-pr-30827
+pr-31229
```

Comment:
Why this change is needed here?

Reply:
to catch changes in docker related files